engine/src/communication/messages/GPUComponentMessage.h

```diff
@@ -189,71 +157,78 @@ class GPUComponentMessage : public GPUMessage {
 		const Address::MetaData & address_metadata,
 		const std::vector<ColumnTransport> & columns_offsets,
 		const std::vector<const char *> & raw_buffers) {  // gpu pointer
```

Comment:
We want to use rmm::device_buffer instead of raw_buffers that way we only allocate once and we never have to make a copy if we use the right constructors or factories

Reply:
ok!